arangodb-net-standard.Test/DatabaseApi/DatabaseApiClientTest.cs

```diff
@@ -187,5 +189,25 @@ public async Task GetCurrentDatabaseInfoAsync_ShouldThrow_WhenDatabaseDoesNotExi
             Assert.Equal(HttpStatusCode.NotFound, apiError.Code);
             Assert.Equal(1228, apiError.ErrorNum);
         }
+
+        [Fact]
+        public async Task DeleteDatabaseAsync_ShouldSucceed()
+        {
+            var response = await _client.DeleteDatabaseAsync(_fixture.DeletableDatabase);
```

Comment:
Can we keep using `_fixture.DatabaseClientSystem`? It's not obvious here which database is used. Or maybe rename `_client` to `_clientSystem` or something to make it more explicit?